openscap_daemon/rest_api.py

```diff
@@ -0,0 +1,370 @@
+# Copyright 2015 Red Hat Inc., Durham, North Carolina.
+# All Rights Reserved.
+#
+# openscap-daemon is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation, either version 2.1 of the License, or
+# (at your option) any later version.
+#
+# openscap-daemon is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+
+# You should have received a copy of the GNU Lesser General Public License
+# along with openscap-daemon.  If not, see <http://www.gnu.org/licenses/>.
+#
+# Authors:
+#   Mario Vazquez <mavazque@redhat.com>
+
+import threading
+import json
+import os
+
+from datetime import datetime
+from flask import Flask, request
```

Comment:
`request` is a pretty generic name, IMO let's just `import flask` and avoid shadowing `request`.

Reply:
request is needed in order to capture json data sent by clients, if we don't import it, call to `request.get_json()` will fail.

Reply:
yes but we could `import flask` and then use `flask.request`.

Reply:
@mpreisler Just to double check. 

I've searched for `request` lib and it seems that it doesn't exist. 

Note that the import is `request` not `requests`. As cipherboy said, importing request from flask is a common practice afaik.

Please, confirm if we should not import `request` and rewrite its calls to `flask.request`. 

Thanks!

Reply:
if it's a common pattern keep it as it is